bootstrap/sql/migrations/native/1.6.0/mysql/schemaChanges.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE data_quality_data_time_series
+ADD COLUMN id VARCHAR(36) GENERATED ALWAYS AS (json ->> '$.id') STORED NOT NULL,
+ADD CONSTRAINT UNIQUE (id);
```

Comment:
It will be good if we can add index for the new column

Reply:
@sonika-shah the table already has an indices on fqn and ts (since most read would be performed filtered on these 2 values).

Reply:
@teddy, it will be okay currently . but  later on, if we add any queries based on the ‘id’ column, it would slow down the operation, even if its a simple insert/delete/update sql statement